modules/aggregation-layers/src/utils/grid-aggregation-utils.js

```diff
@@ -35,6 +38,27 @@ export function getBoundingBox(attributes, vertexCount) {
 }
 /* eslint-enable max-statements */
 
+export function alignBoundingBox(boundingBox, gridOffset, coordinateSystem, viewport) {
```

Comment:
How come the cell size is named `gridOffset`?

Reply:
This is how it is in CPU aggregation code (hexagon and cpugridlayer), there are too many references of `gridOffset`, will make the name change in a different PR.